cpp/cmake/thirdparty/CUDF_GetRMM.cmake

```diff
@@ -16,17 +16,10 @@
 
 function(find_and_configure_rmm VERSION)
 
-    # If building against a local RMM source repo without installing,
-    # set the RMM_HOME and RMM_ROOT environment variables to the source
-    # and binary dirs, i.e. the git repo root and build dir, respectively.
-    if(ARGC GREATER 2 AND (ARGV1 AND ARGV2))
-        set(BUILD_TESTS OFF)
-        set(BUILD_BENCHMARKS OFF)
-        add_subdirectory("${ARGV1}" "${ARGV2}" EXCLUDE_FROM_ALL)
-        return()
-    endif()
-
-    # Alteratively, set `-DCPM_rmm_SOURCE=/path/to/rmm` in the CMake configure flags.
+    # Consumers have two options for local source builds:
+    # 1. Pass `-D CPM_rmm_SOURCE=/path/to/rmm` to build a local RMM source tree
+    # 2. Pass `-D CMAKE_PREFIX_PATH=/path/to/rmm/build` to use an existing local
+    #    RMM build directory as the install location for find_package(rmm)
```

Comment:
Is there a reason we can't use the typical `find_package` convention of `<PackageName>_ROOT` environment variable to find it as well?

Reply:
I didn't add it to the comment, but I think that should work too. I'd rather not modify env vars during the CMake configure process, so instead I'm appending `rmm_BINARY_DIR` in the conditional at the bottom of this function.

Reply:
No I mean a user building libcudf could do `rmm_ROOT=/path/to/rmm/build cmake ...`. Unless I'm misinterpreting this comment somehow.

Reply:
Yeah they can still do that.